transport/client.go

```diff
@@ -311,7 +315,7 @@ func Dial(address string, tr *Transport) (*tls.Conn, error) {
 
 	for _, chain := range state.VerifiedChains {
 		for _, cert := range chain {
-			revoked, ok := revoke.VerifyCertificate(cert)
```

Comment:
1) no. please take into consideration kubernetes, etcd which use cert auth. It is much easier to update a local CRL list rather than reconfigure whole cluster.
2) this resolves the 1st problem
3) well... at least we can automatically fetch public CRL list into fs and use it when there are some connectivity problems with CRL server.

Now I understand what you want me to implement. Will do soon.